main.py

```diff
@@ -1,13 +1,144 @@
+# std libraries
```

Comment:
These comments are extraneous, there's a predefined ordering for these and that's what isort uses.